packages/flutter/lib/src/widgets/notification_listener.dart

```diff
@@ -60,13 +60,13 @@ abstract class Notification {
   /// for each ancestor of a particular type).
   @protected
   @mustCallSuper
+  @Deprecated(
```

Comment:
I'd like to leave this deprecated just in case. Would it be appropriate to add a code snippet for how to replace this, or I could add some sort of "dispatch slowly"

Reply:
PTAL, added a code snippet here. Could also add it to a migration guide